src/ocl/convolutionocl.cpp

```diff
@@ -65,6 +65,7 @@ MIOPEN_DECLARE_ENV_VAR(MIOPEN_DEBUG_CONV_FFT)
 MIOPEN_DECLARE_ENV_VAR(MIOPEN_DEVICE_ARCH)
 MIOPEN_DECLARE_ENV_VAR(MIOPEN_DEBUG_CONV_IMMED_FALLBACK)
 MIOPEN_DECLARE_ENV_VAR(MIOPEN_DEBUG_COMPILE_ONLY)
+MIOPEN_DECLARE_ENV_VAR(MIOPEN_DUMP_TENSOR_PATH)
```

Comment:
Please rename to `MIOPEN_CHECK_NUMERICS_DUMP_TENSOR_PATH` and add documentation to https://github.com/ROCmSoftwarePlatform/MIOpen/blob/develop/doc/src/DebugAndLogging.md#numerical-checking

Reply:
will do